src/superqt/switch/_toggle_switch.py

```diff
@@ -0,0 +1,287 @@
+from __future__ import annotations
+
+from typing import overload
+
+from qtpy import QtCore, QtGui
+from qtpy import QtWidgets as QtW
+from qtpy.QtCore import Property, Qt
+
+
+class QStyleOptionToggleSwitch(QtW.QStyleOptionButton):
+    def __init__(self):
+        super().__init__()
+        self.margin = 2
+        self.on_color = QtGui.QColor("#4D79C7")
+        self.off_color = QtGui.QColor("#909090")
+        self.handle_color = QtGui.QColor("#d5d5d5")
```

Comment:
Hm. Is there a way to integrate it with qss styles (like for napari). Especially for toggling theme?

Reply:
He made them Qproperties… which I believe means you can style with qss

Reply:
Yep: https://doc.qt.io/qt-6/stylesheet-syntax.html#setting-qobject-properties